WORKSPACE

```diff
@@ -3,12 +3,7 @@
 workspace(
     name = "cockroach",
     managed_directories = {
-       "@npm": [
-          "pkg/ui/node_modules",
-          "pkg/ui/workspaces/cluster-ui/node_modules",
-          "pkg/ui/workspaces/db-console/node_modules",
-          "pkg/ui/workspaces/db-console/src/js/node_modules",
-       ],
+       "@npm": ["pkg/ui/node_modules"],
```

Comment:
Bazel don't support yarn workspaces so it cannot properly manage multiple `node_modules` for single workspace.
Packages inside `pkg/ui/workspaces/cluster-ui` and `pkg/ui/workspaces/db-modules` are included in builds with help of `filegroup`s.